cogboard-webapp/src/components/UserLogin.js

```diff
@@ -88,6 +96,24 @@ const UserLogin = () => {
           </Button>
         </StyledFieldset>
       </AppDialog>
+      <SnackbarWithVariant
```

Comment:
I think we can handle this other way, because we have almost the same components. We should provide more data to the redux store on login/logout and then read this data ad pass to snackbar. Just like we have error message now, maybe we should transform it to statusMessage and keep it there. It is just a quick idea, so make a research on that.

Reply:
Definitely worth considering in the near future.